docs/cite.md

```diff
@@ -0,0 +1,35 @@
+# How to Cite Casanovo
+
+When using Casanovo in your research, please cite the relevant scientific publications to acknowledge the work and contributions behind the tool.
+Below, you will find detailed information on how to cite Casanovo, including citations for its various versions and functionalities.
+
+### Main Reference: Casanovo v4.x
+
+For general use of Casanovo, please cite the following paper:
+
+Yilmaz, M., Fondrie, W. E., Bittremieux, W., Melendez, C.F., Nelson, R., Ananth, V., Oh, S. & Noble, W. S. Sequence-to-sequence translation from mass spectra to peptides with a transformer model. in *Nature Communications* **15**, 6427 (2024). [doi:10.1038/s41467-024-49731-x](https://doi.org/10.1038/s41467-024-49731-x)
+
+### Casanovo v2.x: Spectrum Transformer Neural Network
```

Comment:
I think we should ask people to cite the earlier work, to establish priority.  In general, I think people should always include the ICML paper, plus one or more additional papers as needed. I propose removing the subheaders and  just saying "For general use of Casanovo, please cite the following paper:" for the ICML paper, and then "In addition, you may wish to cite one or more of these additional publications, depending on relevance to your work."  Then make a bullet list and, for each one, include a sentence or two describing the main result.